spec/models/document_capture_session_spec.rb

```diff
@@ -2,7 +2,15 @@
 
 RSpec.describe DocumentCaptureSession do
   let(:doc_auth_response) do
-    DocAuth::Response.new(
+    Class.new(DocAuth::Response) do
+      def doc_auth_success?
```

Comment:
What's the benefit of doing this instead of something like
```
before do 
    allow(doc_auth_response).to receive(:doc_auth_success?).and_return(true)
end
```

(I'm not against this approach, I'm just curious.)

Reply:
> What's the benefit of doing this instead of something like
> 
> ```
> before do 
>     allow(doc_auth_response).to receive(:doc_auth_success?).and_return(true)
> end
> ```
> 
> (I'm not against this approach, I'm just curious.)

+1 ...  i agree w/ @eileen-nava  ... it's a more common approach throughout the codebase

Reply:
@amirbey , @eileen-nava , that's true, more familiar with subclassing, since did a lot of Java. 😺